monitoring/cgroup.go

```diff
@@ -54,10 +54,21 @@ func (r cgroupChecker) Name() string {
 // Check verifies existence of cgroup mounts given in r.cgroups.
 // Implements health.Checker
 func (r cgroupChecker) Check(ctx context.Context, reporter health.Reporter) {
+	err := r.check(ctx, reporter)
+	if err != nil && !trace.IsNotFound(err) {
+		reporter.Add(NewProbeFromErr(r.Name(), "failed to validate cgroup mounts", err))
+		return
+	}
+	if reporter.NumProbes() != 0 {
```

Comment:
Shouldn't this go before `r.check`, right at the beginning of the function? To quit early and avoid cases like the one we saw yesterday when an empty probe resulted in a failed check. Just to be on a safe side.